corehq/apps/cloudcare/static/cloudcare/js/form_entry/fullform-ui.js

```diff
@@ -286,6 +286,17 @@ hqDefine("cloudcare/js/form_entry/fullform-ui", function () {
             return self.currentIndex() !== "0" && self.currentIndex() !== "-1" && !self.atFirstIndex();
         });
 
+        self.erroredLabels = ko.computed(function () {
+            var questions = getQuestions(self);
+            var erroredLabels = {};
+            for (var i = 0; i < questions.length; i++) {
+                if (questions[i].isLabel && !questions[i].isValid()) {
+                    erroredLabels[getIx(questions[i])] = "ok";
```

Comment:
why not just make this null or a blank string?

Reply:
Or make this function return an array, since it doesn't look like the values are used.

HQ uses a lot of [underscore](https://underscorejs.org/) to keep code concise. This function could be something like `return _.map(_.filter(getQuestions(self), function (q) { return q.isLabel && !q.isValid(); }), function (q) { return getIx(q); })` (but spaced better so it's more readable)

Reply:
> why not just make this null or a blank string?

today mobile answers all labels as `OK` and though not ideal it would be nice to be consistent here. 

> make this function return an array

I was mostly thinking about future iterations of this feature for non-label questions. For labels, we don't end up using the value though if in future we want to validate any non label questions with a answer change, we would need the formplayer endpoint to support a hashmap in order to pass on realtime values for a question.